index.html

```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="http://s3-us-west-1.amazonaws.com/fec-kickstarter-campaign-module/styles.css">
     <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/updates-and-comments/Updates/styles.css">
     <link rel="stylesheet" href="http://www.samgetlan.com/community/styles.css">
-    <title>NotAirBnB</title>
+    <title>QuickStarter</title>
```

Comment:
Is this static HTML file used at all?

Reply:
This file is not used for sever-side rendering but I'm keeping it as a reference.

Reply:
OK, but if you're no longer using it you should remove it from git tracking.